python/rmm/docs/guide.md

```diff
@@ -67,6 +67,38 @@ host:
 array([1., 2., 3.])
 ```
 
+## Prefetching DeviceBuffers Backed by Managed Memory
+
+CUDA Unified Memory, also known as managed memory, can be allocated using
+rmm.ManagedMemoryResource explicitly, or by calling `rmm.reinitialize` with
+`managed_memory=True`.
+
+A DeviceBuffer backed by managed memory may be prefetched to a specified
+device:
+
+```python
+>>> import rmm
+>>> rmm.reinitialize(managed_memory=True)
+>>> buf = rmm.DeviceBuffer(size=100)
+>>> buf.prefetch()
+```
+
+The above example prefetches the DeviceBuffer memory to the current CUDA device
+on the stream that the DeviceBuffer last used (e.g. at construction). The
+destination device ID and stream are optional parameters.
+
+```python
+>>> import rmm
+>>> rmm.reinitialize(managed_memory=True)
+>>> from rmm._cuda.stream import Stream
+>>> stream = Stream()
+>>> buf = rmm.DeviceBuffer(size=100, stream)
+>>> buf.prefetch(device=3, stream) # prefetch to device on stream.
```

Comment:
```suggestion
>>> buf = rmm.DeviceBuffer(size=100, stream=stream)
>>> buf.prefetch(device=3, stream=stream) # prefetch to device on stream.
```

We can't have positional arguments after keyword arguments in Python.

Reply:
Right, I knew that. 🤦